src/main/java/seedu/multitasky/model/StorageUserPrefs.java

```diff
@@ -0,0 +1,43 @@
+package seedu.multitasky.model;
+
+//@@author A0132788U
+/**
+ * Represents the default values in UserPrefs object that storage has access to.
+ */
+public interface StorageUserPrefs {
+
+    public String getEntryBookFilePath();
+
+    public void setEntryBookFilePath(String entryBookFilePath);
+
+    /**
+     * The getter and setter for the snapshot file path
+     */
+    public static String getEntryBookSnapshotPath() {
+        return UserPrefs.getEntryBookSnapshotPath();
```

Comment:
https://stackoverflow.com/questions/2223386/why-doesnt-java-allow-overriding-of-static-methods